cola/libtopology/topology_graph.h

```diff
@@ -515,10 +513,10 @@ namespace topology {
         void getTopologyConstraints(std::vector<TopologyConstraint*>* ts) 
         const {
             forEach(
-                    std::bind2nd(
-                        std::mem_fun(&EdgePoint::getBendConstraint),ts),
-                    std::bind2nd(
-                        std::mem_fun(&Segment::getStraightConstraints),ts),
+                    std::bind(
```

Comment:
libtopology/./../libtopology/topology_graph.h|519 col 37| warning: 'std::mem_fun1_t<_Ret, _Tp, _Arg> std::mem_fun(_Ret (_Tp::*)(_Arg)) [with _Ret = void; _Tp = topology::EdgePoint; _Arg = vector<topology::TopologyConstraint*>*]' is deprecated: use 'std::mem_fn' instead [-Wdeprecated-declarations]
||   519 |                         std::mem_fun(&EdgePoint::getBendConstraint),ts),
||       |                         ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/13/bits/stl_function.h|1395 col 5| note: declared here
||  1395 |     mem_fun(_Ret (_Tp::*__f)(_Arg))
||       |     ^~~~~~~
libtopology/./../libtopology/topology_graph.h|518 col 33| warning: 'std::binder2nd<_Operation> std::bind2nd(const _Operation&, const _Tp&) [with _Operation = mem_fun1_t<void, topology::EdgePoint, vector<topology::TopologyConstraint*>*>; _Tp = vector<topology::TopologyConstraint*>*]' is deprecated: use 'std::bind' instead [-Wdeprecated-declarations]
||   518 |                     std::bind2nd(
||       |                     ~~~~~~~~~~~~^
||   519 |                         std::mem_fun(&EdgePoint::getBendConstraint),ts),
||       |                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/13/backward/binders.h|172 col 5| note: declared here
||   172 |     bind2nd(const _Operation& __fn, const _Tp& __x)
||       |     ^~~~~~~
libtopology/./../libtopology/topology_graph.h|521 col 37| warning: 'std::const_mem_fun1_t<_Ret, _Tp, _Arg> std::mem_fun(_Ret (_Tp::*)(_Arg) const) [with _Ret = void; _Tp = topology::Segment; _Arg = vector<topology::TopologyConstraint*>*]' is deprecated: use 'std::mem_fn' instead [-Wdeprecated-declarations]
||   521 |                         std::mem_fun(&Segment::getStraightConstraints),ts),
||       |                         ~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/13/bits/stl_function.h|1401 col 5| note: declared here
||  1401 |     mem_fun(_Ret (_Tp::*__f)(_Arg) const)
||       |     ^~~~~~~
libtopology/./../libtopology/topology_graph.h|520 col 33| warning: 'std::binder2nd<_Operation> std::bind2nd(const _Operation&, const _Tp&) [with _Operation = const_mem_fun1_t<void, topology::Segment, vector<topology::TopologyConstraint*>*>; _Tp = vector<topology::TopologyConstraint*>*]' is deprecated: use 'std::bind' instead [-Wdeprecated-declarations]
||   520 |                     std::bind2nd(
||       |                     ~~~~~~~~~~~~^
||   521 |                         std::mem_fun(&Segment::getStraightConstraints),ts),
||       |                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/13/backward/binders.h|172 col 5| note: declared here
||   172 |     bind2nd(const _Operation& __fn, const _Tp& __x)
||       |     ^~~~~~~
|| topology_graph.cpp: In member function 'topology::Segment* topology::EdgePoint::prune(vpsc::Dim)':